src/app/component/Footer/Footer.component.js

```diff
@@ -121,11 +127,33 @@ export class Footer extends PureComponent {
         return <NewsletterSubscription key={ i } />;
     }
 
+    renderCmsBlockWrapper(footer_cms) {
+        return (
+            <div
+              block="Footer"
+              elem="CmsBlockWrapper"
+              mix={ { block: 'Footer', elem: 'Content' } }
+            >
+                <div
+                  block="Footer"
+                  elem="Columns"
+                  mix={ { block: 'ContentWrapper' } }
+                >
+                    <CmsBlock identifier={ footer_cms } />
+                    { this.renderColumn({
+                        ...NEWSLETTER_COLUMN,
+                        mods: { isNewsletter: true }
+                    }) }
+                </div>
+            </div>
+        );
+    }
+
     renderContent() {
         const { footer_content: { footer_cms } = {} } = window.contentConfiguration;
 
         if (footer_cms) {
-            return <CmsBlock identifier={ footer_cms } />;
+            return this.renderCmsBlockWrapper(footer_cms);
```

Comment:
No need to pass this `footer_cms` as an argument, take it from props in the funct

Reply:
Done